### PR TITLE
(Error): Handle server errors in ErrorPage

### LINKED ISSUE #121: Error Page
Create an Error Page to handle 404 and 503

### PR DESCRIPTION
**Pull request by behagoras**
# Description

:sparkles: (Error): Handle server errors in ErrorPage

## Type of change

- [x] This change requires a documentation update
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: pages/_error.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    {
+      code: 404,
+      description: 'We couldn’t find this page.',
+      message: 'Maybe it’s out there, somewhere...',
+      displayPosts: true,
+      displayHome: true,
+    },
+    {
+      code: 503,
+      description: 'Internal Server Error.',
```
Comment (WilliamVelazquez) [inline pages/_error.js]: I would change the message description for something like: **"We are having troubles now"** or something similar

File: components/Layout.js [hunk 2]
```diff
@@ -20,6 +20,7 @@ const Layout = ({ title, children }) => {
         <meta name="viewport" content="initial-scale=1.0, width=device-width" />
         <link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@500;600;700&display=swap" rel="stylesheet" />
         <link href="https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@200;400;600;900&display=swap" rel="stylesheet" />
+        <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" />
```
Comment (WilliamVelazquez) [inline components/Layout.js]: We are using font-awsome?

**Review by WilliamVelazquez — CHANGES_REQUESTED** [review]
Remove Font Awesome and Update Error Messages (Friendly Messages)

**Comment by behagoras** [comment]
Updated code due to code review changes.

**Review by WilliamVelazquez — APPROVED** [review]
Nice!

**Pull request merged**